src/modules/httptrap.c

```diff
@@ -851,6 +855,8 @@ rest_httptrap_handler(mtev_http_rest_closure_t *restc,
       json_object *metrics_obj;
       metrics_obj = json_object_new_object();
 
+      mtevL(nldeb, "HTTPTRAP: Processed %d records\n", cnt);
```

Comment:
The nldeb logging facility is already named debug/httptrap so the extra HTTPTRAP is redundant.

Reply:
done